tests/scripts/pulp_python/test_distribution.sh

```diff
@@ -38,14 +38,22 @@ expect_succ pulp python distribution update \
   --base-path "cli_test_python_distro" \
   --publication "$PUBLICATION_HREF"
 
-expect_succ curl "$curl_opt" --head --fail "$PULP_BASE_URL/pulp/content/cli_test_python_distro/simple/"
-
 if [ "$(pulp debug has-plugin --name "python" --min-version "3.4.0.dev")" = "true" ]
```

Comment:
```suggestion
if pulp debug has-plugin --name "python" --min-version "3.4.0.dev"
```

This command was written for this purpose.